src/hdtSkinnedMesh/hdtSkinnedMeshWorld.cpp

```
@@ -94,7 +94,7 @@ namespace hdt
 			for (auto j : i->m_constraints)
 				addConstraint(j->m_constraint, true);
 
-		for (int i = 0; i < system->m_constraints.size(); ++i)
+		for (size_t i = 0; i < system->m_constraints.size(); ++i)
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)